apps/sms/style/root.css

```diff
@@ -177,3 +177,11 @@ menu, nav, section, video, audio, canvas, progress, meter, time {
   margin: 0;
 }
 
+/* Dialogs */
+body.dialog-animating {
```

Comment:
Agree with you, we could use this css to perform the task whenever we want, not just in the body tag.